ReplayKitExample.xcodeproj/project.pbxproj

```diff
@@ -363,6 +407,63 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		24748EF3252B912300EEBFDD /* Sign Frameworks */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Sign Frameworks";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "find \"${CODESIGNING_FOLDER_PATH}\" -name '*.framework' -print0 | while read -d $'\\0' framework \ndo \n    codesign --force --deep --sign \"${EXPANDED_CODE_SIGN_IDENTITY}\" --preserve-metadata=identifier,entitlements --timestamp=none \"${framework}\" \ndone\n";
```

Comment:
Did Xcode generate this script or did we add it?

Reply:
I did. There is a known issue with code signing SPM binary frameworks as of Xcode 12.2 still.

https://forums.swift.org/t/swiftpm-binarytarget-dependency-and-code-signing/38953/41
https://bugs.swift.org/browse/SR-13343